corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -0,0 +1,78 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<!-- B5 TODO: Rewrite this modal to use B5 built-in utility -->
+<div
+  class="modal fade define-case-type-modal"
+  tabindex="-1"
+  role="dialog"
+  id="define-case-type-modal"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button
+          type="button"
+          class="close"
+          data-dismiss="modal"
+          aria-label="Close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">
+          <i class="fa-solid fa-list-check"></i> {% trans "Case List" %}
+        </h4>
+      </div>
+      <div class="modal-body">
+        <h4>
+          <strong
+            >{% trans "Please define a case type for your case list." %}</strong
+          >
+        </h4>
+        <div class="form-group" id="new_case_type_form_group">
+          <label class="control-label">
+            {% trans "Case Type" %}
+            {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
+              <span
+                class="hq-help-template"
+                data-title="{% trans "Case Type" %}"
+                data-content="{% trans "The name for the type of case you are tracking (e.g. “pregnancy,” “farm,” “household,” etc.). All cases you register in this case list will be this type of case. Other case lists with the same case type will use the same set of case(s)." %}"
+              ></span>
+            {% else %}
+              <span
+                class="hq-help-template"
+                data-title="{% trans "Case Type" %}"
+                data-content="{% trans "The name for the type of case you are tracking (e.g. “pregnancy,” “farm,” “household,” etc.). All cases you register in this case list will be this type of case, and only cases of this type will show in the case list. Other case lists with the same case type will use the same set of cases." %}"
+              ></span>
+            {% endif %}
+          </label>
+          <div class="new-case-type-dropdown-container">
+            <select
+              class="form-control new-case-type-dropdown"
+              id="new_case_type"
+              name="new_case_type"
+            >
+              <option value=""></option>
+            </select>
+            <span class="help-block" id="new_case_type_help">
```

Comment:
the span here should also be a div

Reply:
Fixed together in d54d9fe549d150d7f3a36a2b18833563b0237912